### PR TITLE
Rustify nvpairs.h for Tedsta

### PR DESCRIPTION
**Pull request by stratact**

### CODE REVIEW

File: filesystem/apps/zfs/nvpair.rs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    Int16Array,
+    Uint16Array,
+    Int32Array,
+    Uint32Array,
+    Int64Array,
+    Uint64Array,
+    StringArray,
+    Hrtime,
+    Nvlist, // TODO: What to name this ?
+    NvlistArray,
```
Comment (tedsta) [inline filesystem/apps/zfs/nvpair.rs]: Could you change:

`Hrtime` -> `HrTime`
`Nvlist` -> `NvList`
`NvListArray` -> `NvListArray`

Seems more consistent with the rest of the naming

**Comment by tedsta** [comment]
:+1: I like it

**Comment by stratact** [comment]
Glad I could help. :smile:

**Comment by stratact** [comment]
All checks have pass, are we good to go from here?

**Comment by Ticki** [comment]
Rustastic!

**Pull request merged**